pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.3</version>
+    <version>1.580.1</version>
```

Comment:
As a rule I always use: "Upgrade Jenkins baseline to the next less restrictive LTS that includes the API that I want to use". So the "less restrictive LTS" is always the XXX.1. This allow to use the plugin in any revision of the first LTS.